cert_agent/tasks/main.yml

```diff
@@ -40,7 +40,12 @@
 
 - name: Set up virtualenv postactivate scripts
   become_user: "{{ cert_agent_app_user }}"
-  template: src=postactivate dest={{cert_agent_venv_dir}}/bin/postactivate mode=0700 owner={{ cert_agent_app_user }} group={{ cert_agent_app_user }}
+  template:
```

Comment:
Can we use the dict alternative?

```
template:
  src: postactivate
  dest: "{{cert_agent_venv_dir}}/bin/postactivate"
  mode: 0700
  owner: "{{ cert_agent_app_user }}"
  group: "{{ cert_agent_app_user }}"
```

It gets colored nicely in PyCharm this way.